integration-core/src/test/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeTest.java

```diff
@@ -174,9 +194,6 @@ public void testSendMessageUnexpectedException() throws JsonProcessingException,
     instance.setOptionalProperties(OPTIONAL_PROPERTIES);
 
     List<Message> result = bridge.sendMessage(instance, INTEGRATION_USER, "message");
-
-    assertNotNull(result);
-    assertTrue(result.isEmpty());
   }
 
```

Comment:
Please add a test case to cover the HTTP 500 status code. Preferably you can return more than one exception with different status codes.